### PR TITLE
Loi service

### PR DESCRIPTION
**Pull request by jyotishk-eGov**

### CODE REVIEW

File: backend/loi-service/Letter-Of-Indent (LOI).postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (rushang7-eGov) [inline backend/loi-service/Letter-Of-Indent (LOI).postman_collection.json:1]: Verify if this collection has all the apis. If not, add them.

File: backend/loi-service/src/main/java/org/egov/works/config/LOIConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+@Setter
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Import({TracerConfiguration.class})
+@Component
+public class LOIConfiguration {
+    //Workflow Config
+    @Value("${egov.workflow.host}")
```
Comment (rushang7-eGov) [inline backend/loi-service/src/main/java/org/egov/works/config/LOIConfiguration.java:19]: Workflow host config has been repeated below. Remove it from here. 
Check if there are any other repeatitions.

File: backend/loi-service/src/main/java/org/egov/works/config/LOIConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    private String workflowHost;
+    @Value("${egov.workflow.transition.path}")
+    private String workflowTransitionPath;
+    @Value("${egov.workflow.processinstance.search.path}")
+    private String wfProcessInstanceSearchPath;
+    @Value("${egov.workflow.businessservice.search.path}")
+    private String wfBusinessServiceSearchPath;
+
+    @Value("${workflow.loi.business.service.name}")
```
Comment (rushang7-eGov) [inline backend/loi-service/src/main/java/org/egov/works/config/LOIConfiguration.java:68]: These 2 configs are present above.

**Review by rushang7-eGov — COMMENTED** [review]
Add CHANGELOG.md. Mention the remaining todos of the project.

File: backend/loi-service/src/main/java/org/egov/works/web/models/LetterOfIndentRequestWorkflow.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ */
+@Validated
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2022-08-04T15:05:28.525+05:30")
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class LetterOfIndentRequestWorkflow {
```
Comment (rushang7-eGov) [inline backend/loi-service/src/main/java/org/egov/works/web/models/LetterOfIndentRequestWorkflow.java:22]: Use workflow class from digit-models library.

File: backend/loi-service/src/main/java/org/egov/works/util/LOIConstants.java [hunk 5]
```diff
@@ -0,0 +1,6 @@
+package org.egov.works.util;
+
+public class LOIConstants {
+    public static final String MDMS_MODULE_NAME = "tenant";
```
Comment (rushang7-eGov) [inline backend/loi-service/src/main/java/org/egov/works/util/LOIConstants.java:4]: Rename to MDMS_TENANT_MODULE_NAME. Same apples to the other variable. 
There might be other module names for which we could be calling mdms from the loi-service.

Comment (jyotishk-eGov) [reply backend/loi-service/src/main/java/org/egov/works/util/LOIConstants.java:4]: Done.

File: backend/loi-service/src/main/java/org/egov/works/web/models/LetterOfIndentRequestWorkflow.java (hunk 4 above)
Comment (jyotishk-eGov) [reply backend/loi-service/src/main/java/org/egov/works/web/models/LetterOfIndentRequestWorkflow.java:22]: Could not find the Workflow clas in Digit-models library

File: backend/loi-service/src/main/java/org/egov/works/config/LOIConfiguration.java (hunk 2 above)
Comment (jyotishk-eGov) [reply backend/loi-service/src/main/java/org/egov/works/config/LOIConfiguration.java]: Done

File: backend/loi-service/Letter-Of-Indent (LOI).postman_collection.json (hunk 1 above)
Comment (jyotishk-eGov) [reply backend/loi-service/Letter-Of-Indent (LOI).postman_collection.json:1]: Added create, modify and search apis

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**